data/remnant missions.txt

```diff
@@ -493,5 +597,771 @@ mission "Remnant: Technology Available"
+			choice
+				`	"Threats?"`
+					goto SalvThreats
+				`	"Interesting."`
+			`	"Yes, it is, isn't it? So many new things out there to discover and learn about."`
+				goto SalvEnd
+			label SalvThreats
+			`	"Yes, sometimes salvage that is brought in is dangerous. System Cores come with fleets of micro-bots, for instance. If they aren't de-activated properly, they keep trying to rebuild their surroundings in the form of the last ship they were configured to serve. And while it hasn't happened yet, we are worried that someday we might run into someone who builds traps into their equipment, or uses nanobots that could run amok. We haven't seen any of that yet, but our science-fiction stories are filled with tales of nanotech apocalypses and things like that."`
+			label SalvEnd
+			`	"Well, I need to get back to work. That last battle put us behind schedule on our repairs. Have a good day!" With a final flourish that you think is the equivalent of an exclamation mark, Tali strides out of the outfitters, heading towards another landing bay.`
+
+event "Remnant Salvage Available"
+	outfitter "remnant"
+		"Generator (Furnace Class)"
+		"Generator (Candle Class)"
+		"Triple Plasma Core"
+		"Small Heat Shunt"
+		"Large Heat Shunt"
+		"Systems Core (Small)"
+		"Thruster (Asteroid Class)"
+		"Thruster (Planetary Class)"
+		"Steering (Asteroid Class)"
+		"Steering (Planetary Class)"
+		"Thruster (Comet Class)"
+		"Steering (Comet Class)"
+		"Korath Fire-Lance"
+	
+
+mission "Remnant: Expanded Horizons Quarg 1"
+	name "Expanded Horizons visiting the Quarg"
+	description "Take Remnant researchers to see some Quarg."
+	passengers 3
+	blocked "This mission requires bunks for three passengers."
+	source
+		government "Remnant"
+	destination "Wayfarer"
+	to offer
+		has "Remnant: Learning Sign 2: done"
+		has "First Contact: Quarg: offered"
+		random < 75
+	on offer
+		conversation
+			`As you relax in the cafeteria on <planet> you are approached by young Remnant. Judging by the hand-held sensors and tools hanging off her belt, you suspect she is a researcher or an engineer.`
+			`	"Hello Captain <last>, My name is Dawn, and I have been spending the past few days skimming the archive you retrieved. I wanted to ask you a few questions. May I?"`
+			choice
+				`"Sure, pull up a chair."`
+					goto AnswerQuestions
+				`"Not right now."`
+					defer
+				`"I'm not interested."`
+					decline
+			label "AnswerQuestions"
+			`	She quickly slides into the chair opposite and pulls out a datapad. "Firstly, our histories mention how humanity met a race called the 'Quarg' and had some ongoing dialogs with them, but not much beyond that. Now here in this history that you brought us, it says that humanity has started exchanging technology with them! Is this true?" She looks at you intently. You sign affirmatively, and she excitedly continues. "Could you tell me about them? An alien race that actually exchanges tech peacefully with humanity is exciting!"`
```

Comment:
You are correct. I think it has been established that the Quarg are (usually) willing to give humanity information (history of the galaxy, scientific info, etc), but they don't give up tech. I've rewritten the section:

She quickly slides into the chair opposite and pulls out a datapad. "Firstly, our histories mention how humanity met a race called the 'Quarg' and had some ongoing dialogs with them, but not much beyond that. Now here in this history that you brought us, it says that humanity has started exchanging technology with them! Is this true?" She looks at you intently. You sign negatively, and she dejectedly continues. "Could you tell explain about them, then?" You reply that the Quarg have allowed humanity to live near them, and that the Quarg are usually willing to answer questions, including scientific ones. You continue, elaborating that while the Quarg don't seem to mind having an innovative shipyard studying them as best it can, they don't actually share their technology with anyone."